package.json

```diff
@@ -79,5 +81,18 @@
     "*.js": [
       "eslint --format=codeframe --rulesdir='./scripts/eslint_rules'"
     ]
+  },
+  "jest": {
+    "runner": "jest-runner-mocha",
+    "testMatch": [
+      "<rootDir>/packages/**/test/*.js"
```

Comment:
Should include `codemods/` and `experimental/` (though babel-preset-env will need a bit of work to work with jest, I imagine).

Reply:
Just FYI, now, the `experimental` doesn't exists anymore. Only `codemods` must be included here.